src/main.ts

```diff
@@ -494,9 +494,9 @@ export class QueryFeature extends SqlOpsFeature<undefined> {
 			);
 		};
 
-		let initializeEdit = (ownerUri: string, schemaName: string, objectName: string, objectType: string, LimitResults: number): Thenable<void> => {
+		let initializeEdit = (ownerUri: string, schemaName: string, objectName: string, objectType: string, LimitResults: number, queryString: string): Thenable<void> => {
```

Comment:
Should this be optional via the `query? :string`

Reply:
It gets set as undefined if not needed.